changes/14698-macos-desktop-err

```diff
@@ -0,0 +1,2 @@
+- Fixed issue on MacOS with starting Fleet Desktop for the first time. MacOS would return an error
+  if a user is not logged in via the GUI.
```

Comment:
This is easy to miss, but there's a different changelog for `fleetd`, you need to write the changes file in `orbit/changes` instead of here.

Reply:
Derp, thanks! Totally missed that.